tests/sorcha/test_PPCommandLineParser.py

```diff
@@ -7,10 +7,11 @@
 class args:
     def __init__(self, cp, dw, dr, dl, t="testout", u="./", f=False):
         self.p = get_test_filepath("testcolour.txt")
-        self.o = get_test_filepath("testorb.des")
+        self.ob = get_test_filepath("testorb.des")
         self.e = get_test_filepath("oiftestoutput.txt")
         self.c = get_test_filepath("test_PPConfig.ini")
-        self.u = u
+        self.pd = get_test_filepath("baseline_10klines_2.0.db")
+        self.o = u
```

Comment:
How much more work would it be to change the input parameter name from `u` to `o` here?

Reply:
Good call. I fixed throughout the test.